example/common/src/commonMain/kotlin/com/konyaco/fluent/example/App.kt

```diff
@@ -84,7 +71,266 @@ fun App() {
                     ) { it / 6 } with
                     fadeOut(tween(FluentDuration.QuickDuration, easing = FluentEasing.FastInvokeEasing))
         }) {
-            navs[it].content()
+            selectedItemWithContent.content?.invoke()
```

Comment:
Using `it.content?.invoke()` can fix the page flickering issue. By the way, in what situation would the `content` be `null`?

Reply:
keep the same behavior in the `WinUI 3 Gallery`, some NavigationItem can be selected but can't navigate content.
![image](https://user-images.githubusercontent.com/26089739/233855065-4658ae3f-4e16-40f2-a014-1ddab645eb79.png)